tests/e2e/packages/verify_packages_install_deb.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+# This test installs debian packages that are a result of the CI and release builds.
+# It runs '... --version' commands to verify that the binaries are correctly installed
+# and finally uninstalls the packages.
+# For the release packages it tests all versions in the support skew.
+
+LINE_SEPARATOR="*************************************************"
+echo "$LINE_SEPARATOR"
+
+PACKAGES_TO_TEST_K8S="kubectl kubelet kubeadm"
```

Comment:
consider using shell arrays instead of relying on word splitting later:
```suggestion
PACKAGES_TO_TEST_K8S=("kubectl" "kubelet" "kubeadm")
```